pod/annotations.go

```diff
@@ -756,6 +757,9 @@ func PlatformSidecars(annotations map[string]string) ([]PlatformSidecar, error)
 		if args, ok := annotations[SidecarAnnotation(sidecar.Name, "arguments")]; ok {
 			sidecar.ArgsJSON = []byte(args)
 		}
+		if channelDefinitionID, ok := annotations[SidecarAnnotation(sidecar.Name, "channelDefinitionID")]; ok {
```

Comment:
Should this be `channel-definition-id` for consistency with the other annotations?